helpers/decompress/decompress_message.go

```diff
@@ -0,0 +1,29 @@
+package decompress
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+)
+
+func GZipMessageToString(message []byte) (string, error) {
```

Comment:
Could just rename to ParseGZip or even jsut GZip since the package name is decompress so this becomes: decompress.GZip(...